components/layout/index.styl

```diff
@@ -1,12 +1,13 @@
-@import '../../node_modules/nib'
+// Common (includes Nib)
+@import '../../node_modules/artsy-elan/components/lib'
+
+// Components
+@import '../../node_modules/artsy-elan/assets/stylesheets/components'
+@import '../nav'
 @import '../header'
+
 @import '../footer'
 @import '../body'
-@import '../carousel/carousel.styl'
-@import '../carousel/desktop_carousel.styl'
-@import 'button'
-@import '../header/desktop'
+@import '../carousel/carousel'
+@import '../carousel/desktop_carousel'
```

Comment:
Oh I seeee. No that totally makes sense actually. I can wipe those above the fold commits out if makes sense.